From 1bbae89303db5ea13be241b76f8fb574f85dafa1 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Thu, 3 Feb 2022 12:16:45 +0000
Subject: [PATCH 1376/2201] dtoverlays: Correct field sizes in pcie-32bit-dma

Adding the dma-ranges to the overlay missed setting the field sizes,
so the compiler rightly flagged a warning.

https://github.com/raspberrypi/linux/issues/4848

Fixes: ee6a81c85402 "dtoverlay: Reduce size of PCIE IB window in pcie-32-dma overlay"

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/pcie-32bit-dma-overlay.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/overlays/pcie-32bit-dma-overlay.dts b/arch/arm/boot/dts/overlays/pcie-32bit-dma-overlay.dts
index e01c9ceeab08..955703563df7 100644
--- a/arch/arm/boot/dts/overlays/pcie-32bit-dma-overlay.dts
+++ b/arch/arm/boot/dts/overlays/pcie-32bit-dma-overlay.dts
@@ -28,6 +28,8 @@
 			 * so the range ends up being 0-4GB, and the MSI vector
 			 * gets pushed beyond 4GB.
 			 */
+			#address-cells = <3>;
+			#size-cells = <2>;
 			dma-ranges = <0x02000000 0x0 0x00000000 0x0 0x00000000
 				      0x0 0x80000000>;
 		};
-- 
2.17.1

